src/components/SearchSection/SearchSection.js

```diff
@@ -0,0 +1,89 @@
+import { useEffect, useState } from 'react';
+import { BiCurrentLocation } from 'react-icons/bi';
+
+import SearchPlaceModal from 'components/SearchPlaceModal/SearchPlaceModal';
+import { keyAPI } from 'components/config';
+
+import styles from './SearchSection.module.css';
+import 'index.css';
```

Comment:
why we need that import?

Reply:
`import 'index.css';`được import để sử dụng các class chung như `displayItem`, `hideItem`.